samza-rest/src/main/java/org/apache/samza/monitor/MonitorConfig.java

```diff
@@ -42,6 +42,8 @@
 
   private static final String MONITOR_PREFIX = String.format("monitor%s", MONITOR_CONFIG_KEY_SEPARATOR);
 
+  private static final String CONFIG_JITTER_ENABLED = "jitter.enabled";
```

Comment:
Make this config a percent b/w 0 to 100, default to 0, and add that amount of jitter to the scheduling interval.

Also, namespace the config correctly. This is a scheduling related config, so should be scheduling.jitter.percent or something.

Reply:
Done. Added a jitter percent which gets multiplied with random number.